static/js/components.js

```diff
@@ -125,6 +125,28 @@ exports.toggle = function (opts) {
     return prototype;
 };
 
+exports.register_tooltip = function (elem, content, placement) {
+    elem.tooltip({
+        title: content,
+        trigger: 'hover',
+        placement: placement,
+        animation: false,
+    });
+    elem.tooltip('show');
+};
+
+exports.register_title_tooltip = function (elem, user_id) {
```

Comment:
I don't think we want code that is this buddy-list-specific to be in components.  The components piece should just be the stuff that is generic, like `register_tooltip` above.  For code that is custom to the buddy list, we can put that in `buddy_list.js`.

Reply:
I'll move it. It is also used by the pm list by the way, but yeah one or the other.